### PR TITLE
fix: modified voice commands

### LINKED ISSUE #2150: Letters gets over written
**Describe the bug**
The letters after voice command are getting overwritten.

**Expected Behaviour**
It should not get over written it should either be displayed up of the chatlists or below that
** Screen shots **
Screenshot_20190408-223113_SUSIAI

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by Bhavnaharitsa**
Fixes #2150 

Changes:When the user talks, the txtchat is hidden by the voice commands.
Modified it by reducing the size and making it top of txtChat

Screenshots for the change: 
Screenshot_20190413-154452_SUSIAI

### CODE REVIEW

**Comment by Bhavnaharitsa** [comment]
@iamareebjamal Please review this !!

**Comment by Bhavnaharitsa** [comment]
@iamareebjamal Please review this sir!!

**Comment by iamareebjamal** [comment]
Peer reviews

**Comment by Bhavnaharitsa** [comment]
@atm1504 please review this for me!!

File: app/src/main/res/layout/fragment_sttframe.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <android.support.v7.widget.RecyclerView
         android:id="@+id/clickableCommands"
         android:layout_width="match_parent"
-        android:layout_above="@id/linearLayout"
+        android:layout_above="@id/txtChat"
```
Comment (atm1504) [inline app/src/main/res/layout/fragment_sttframe.xml]: What is the need of renaming it?

File: app/src/main/res/layout/item_voice_commands.xml [hunk 2]
```diff
@@ -13,6 +13,6 @@
         tools:text="Open WhatsApp"
         android:id="@+id/voiceCommand"
         android:padding="@dimen/padding_moderate"
-        android:textSize="@dimen/text_size_large"/>
+        android:textSize="@dimen/text_size_medium"/>
```
Comment (atm1504) [inline app/src/main/res/layout/item_voice_commands.xml]: Is just changing the textsize doing the work?

Comment (Bhavnaharitsa) [reply app/src/main/res/layout/item_voice_commands.xml]: Yes it is!

**Comment by atm1504** [comment]
@iamareebjamal I have reviewed it. Its okk from my side.

**Comment by Bhavnaharitsa** [comment]
@iamareebjamal Please review this and merge this!!

**Comment by iamareebjamal** [comment]
Can't merge until CircleCI works

**Comment by Bhavnaharitsa** [comment]
> Can't merge until CircleCI works

Any idea how to check for ci build failure?

**Comment by iamareebjamal** [comment]
It's not even being triggered. Only happening on your PR apparently

**Comment by Bhavnaharitsa** [comment]
> It's not even being triggered. Only happening on your PR apparently

Yes!! Any idea how to see what the problem is?

**Comment by atm1504** [comment]
@batbrain7 @arundhati24 as the ci/circleci is failing to trigger can I make a new pr to solve this issue?

**Pull request merged**